plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -94,9 +94,7 @@ func (r *NodeAuthorizer) Authorize(attrs authorizer.Attributes) (authorizer.Deci
 			return r.authorizeGet(nodeName, configMapVertexType, attrs)
 		case pvcResource:
 			if r.features.Enabled(features.ExpandPersistentVolumes) {
-				if attrs.GetSubresource() == "status" {
```

Comment:
dropping this means a get will fail

Reply:
Ahh, I see now.

Reply:
Unit tests passed :(

Reply:
ah, looks like only integration tests were added for this.